controller/ApplicationsController.js

```diff
@@ -0,0 +1,49 @@
+const mongoose = require("mongoose");
+const CustomError = require("../utils/customError");
+const responseHandler = require("../utils/responseHandler");
+// Application model
+const Applications = require("../models/applications");
+const Organizations = require("../models/organizations");
+
+exports.getAllApplications = async (req, res, next) => {
+  try {
+    const { orgId } = req.params;
```

Comment:
The `organizationId` is actually decoded from the `organizationToken` and is available on `req.token` not `req.params`. The proper path for this has no params as stated in the task sheet. Also, check the swagger docs to confirm this